lib/cli/index.js

```diff
@@ -1,27 +1,61 @@
-var cli  = require('ember-cli/lib/cli');
-var path = require('path');
-var fs   = require('fs');
-
-module.exports = function (options) {
-  process.stdout.write = (function(write) {
-    return function(string, encoding, fd) {
-      if (/version:/.test(string) || /warning:/.test(string)) {
-        return;
-      }
-      string = string.replace(/ember-cli(?!.com)/g, 'angular-cli');
-      string = string.replace(/ember(?!-cli.com)/g, 'ng');
-      write.apply(process.stdout, arguments)
+const cli = require('ember-cli/lib/cli');
+const path = require('path');
+
+
+// This file hooks up on require calls to transpile TypeScript.
+const fs = require('fs');
+const ts = require('typescript');
+const old = require.extensions['.ts'];
+
+require.extensions['.ts'] = function(m, filename) {
```

Comment:
`ts-node` was including too many definition files and it was showing duplicates.